server2/Arcadia.Assistant/Arcadia.Assistant.Notifications.Contracts/Models/NotificationMessage.cs

```diff
@@ -8,6 +8,9 @@ public class NotificationMessage
         [DataMember]
         public string NotificationTemplate { get; set; } = string.Empty;
 
+        [DataMember]
+        public string Title { get; set; } = string.Empty;
```

Comment:
what's the difference between title and subject?

Reply:
We discussed fields before. As i remember, title is a short theme for notification. As a channel name.